db/bigtable_eth1.go

```diff
@@ -2102,8 +2102,10 @@ func (bigtable *Bigtable) GetAddressInternalTableData(address []byte, search str
 
 	names := make(map[string]string)
 	for _, t := range transactions {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
+		if t != nil {
```

Comment:
A check similar to this one is also required here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/cf3d27e3ce7d3f94bba0a78e1387afe7fbb0d602/handlers/api_eth1.go#L506
(uses `GetEth1ItxForAddress` too)

Also, here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/cf3d27e3ce7d3f94bba0a78e1387afe7fbb0d602/handlers/api_eth1.go#L424
(uses `GetEth1TxForAddress` which is similar)